music/apiserver_funcs.go

```
@@ -804,3 +809,59 @@ func APIshow(conf *Config, router *mux.Router) func(w http.ResponseWriter, r *ht
+		}
+
+		log.Printf("APIsidecar: received /sidecar request (command: %s) from %s.\n",
+			sp.Command, r.RemoteAddr)
+
+		var resp = SidecarResponse{
+			Status: 101,
+		}
+
+		defer func() {
+			w.Header().Set("Content-Type", "application/json")
+			err := json.NewEncoder(w).Encode(resp)
```

golangci-lint (1.62.2)

SA1026: trying to marshal unsupported type func(req *net/http.Request, via []*net/http.Request) error, via x.Sidecars[k].Api.Client.CheckRedirect

(staticcheck)